packages/react/src/components/Badge/Badge.tsx

```diff
@@ -64,44 +64,31 @@ const paragraphSizeMap: {
  * </Badge>
  * ```
  */
-export const Badge = forwardRef<HTMLSpanElement, BadgeProps>(
-  (
-    {
-      color = 'accent',
-      size = 'md',
-      placement = 'top-right',
-      overlap = 'rectangle',
-      count,
-      maxCount,
-      children,
-      className,
-      ...rest
-    },
-    ref,
-  ) => {
-    return (
-      <div className={cl('ds-badge__wrapper', className)}>
-        {children}
-        <Paragraph asChild variant='short' size={paragraphSizeMap[size]}>
-          <span
-            className={cl('ds-badge', count && 'ds-badge--count')}
-            ref={ref}
-            data-size={size}
-            data-color={color}
-            {...(children
-              ? {
-                  'data-placement': placement,
-                  'data-overlap': overlap,
-                }
-              : {})}
-            {...rest}
-          >
-            {maxCount && count && count > maxCount ? `${maxCount}+` : count}
-          </span>
-        </Paragraph>
-      </div>
-    );
+export const Badge = forwardRef<HTMLSpanElement, BadgeProps>(function Badge(
+  {
+    className,
+    color = 'accent',
+    count,
+    maxCount = Infinity,
```

Comment:
Should also write about this default in the jsdoc

Reply:
Mm, I think it will be confusing as not everyone is familiar with `Infinity` so I've instead removed the default and added a check :)